src/main/java/org/mule/extension/ftp/internal/connection/FtpConnectionProvider.java

```diff
@@ -135,6 +139,23 @@ public String getWorkingDir() {
   @DisplayName("Enable Remote Verification")
   private boolean remoteVerificationEnabled = true;
 
+  /**
+   * Set the control encoding to use in the command channel with the remote server. This is does NOT set the encoding for the content of the files transferred.
+   * <p>
+   * Known control encodings:
+   * <ul>
+   *     <li>ISO-8859-1</li>
+   *     <li>UTF-8</li>
+   * </ul>
+   */
+  @Parameter
+  @Optional(defaultValue = "ISO-8859-1")
+  @Placement(tab = ADVANCED_TAB, order = 3)
+  @OfValues(controlEncodingValueProvider.class)
+  @Summary("Set the control encoding (for example UTF-8) to use in the command channel with the remote server. This is does NOT set the encoding for the content of the files transferred.")
```

Comment:
I would either say control encoding or command channel, both seems redundant.

Reply:
This is does NOT set the encoding for the content of the files transferred.
->
This does NOT set the encoding for the content of the files to be transferred.